charts/fpl-case-service/values.preview.template.yaml

```diff
@@ -13,6 +13,7 @@ java:
     SPRING_SECURITY_ENABLED: true
     SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_ISSUERURI: https://forgerock-am.service.core-compute-idam-aat.internal:8443/openam/oauth2/hmcts
     SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_JWKSETURI: https://idam-api.aat.platform.hmcts.net/o/jwks
+    DOCUMENT_MANAGEMENT_GATEWAY_URL: https://gateway-${SERVICE_FQDN}
```

Comment:
doesn't we need to have -ccd? 
The above will result in
gateway-fpl-case-service-pr-618.service.core-compute-preview.internal. Is this correct URL? Can you access it?